CMakeLists.txt

```diff
@@ -54,6 +54,7 @@ if(APPLE)
     find_package(OpenSSL REQUIRED)
 else()
     pkg_search_module(OPENSSL REQUIRED openssl)
```

Comment:
You can do that in the same line, no need to call `pkg_check_modules` a second time. Also, look at #22, which touches this code anyway.

Reply:
You can do it in one line, but in that case, the output from CMake in case of version mismatch is misleading. It complains that package was not found and does not say why (version mismatch).

Reply:
But if you do it in a separate call and do not pass the `REQUIRED` flag, it will just print a message during configure, but not actually fail the configure phase…